src/crypto/ocb.cc

```diff
@@ -260,7 +260,10 @@
 	}
 	static inline block double_block(block b)
 	{
-		const block mask = {135,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1};
+		const block mask = {(int8_t)135,(int8_t)1,(int8_t)1,(int8_t)1,(int8_t)1 \
+					,(int8_t)1,(int8_t)1,(int8_t)1,(int8_t)1 \
+					,(int8_t)1,(int8_t)1,(int8_t)1,(int8_t)1 \
+					,(int8_t)1,(int8_t)1,(int8_t)1};
```

Comment:
Surely you can just replace `135` with `-121` instead of adding all these casts?